BaseStation/PTZCamera/Camera_Control.cs

```diff
@@ -10,6 +10,9 @@
 
 namespace HuskyRobotics
 {
+    /// <summary>
+    /// This class is just used to test the camera controls.
+    /// </summary>
     class Camera_Control
```

Comment:
Rename to TestCameraControl or something else to indicate that it is a test. Also the naming convention for classes does not use underscores. Also integrate with the rest of the project, so that we don't just have some random Main method.